README.md

```diff
@@ -722,6 +722,62 @@ Req/Bytes counts sampled once per second.
 125k requests in 10.03s, 43.7 MB read
 ```
 
+## More info about how this plugin works
+This plugin caches the resolver's results. The plugin has nothing to do with validating the schema, so if you have a resolver that returns a `String` but in your schema it indicates that the resolver returns an `Int`, the plugin will cache the `String`, and when you call the resolver again, the plugin will return the cached `String` instead of the `Int`. 
+As a result, the query will return a Schema Validation error and not the right result.
+
+
+Here you can find an example of the problem.
+```js
+'use strict'
+
+const fastify = require('fastify')
+const mercurius = require('mercurius')
+const cache = require('mercurius-cache')
+
+const app = fastify({ logger: true })
+
+const schema = `
+  type Query {
+    add(x: Int, y: Int): Int  // <-- this is a Int
+    hello: String
+  }
+`
+
+const resolvers = {
+  Query: {
+    async add (_, { x, y }, { reply }) {
+      reply.log.info('add called')
+
+      return 'not a number' // <-- this is a String
+    }
+  }
+}
+
+app.register(mercurius, {
+  schema,
+  resolvers
+})
+
+
+// cache query "add" responses for 10 seconds
+app.register(cache, {
+  ttl: 10,
+  policy: {
+    Query: {
+      add: true
+      // note: it cache "add" but it doesn't cache "hello"
+    }
+  }
+})
+
+app.listen({ port: 3000, host: '127.0.0.1' })
+```
+
+If you come across this problem, to rectify the data in the cache, follow these instructions:
+
+1. Are you using an **In-Memory** cache? If yes, after the new deployment, the cache will be cleared, and everything will be ok.
```

Comment:
```suggestion
1. If you are you using an in-memory cache, it will be cleared at the next start of the application, so the impact of this issue will be limited
```